addons/mail/static/src/scss/chatter.scss

```diff
@@ -22,6 +22,11 @@
         }
     }
 
+    .o_thread_message_sidebar img:hover {
```

Comment:
is it not simply o_thread_message_avatar ? Or this is because it should not apply to o_thread_message_sidebar_image img ?